mapserver/crdppf.map.mako

```diff
@@ -1056,6 +1056,26 @@ LAYER
         COLOR  157 159 162
     END #STYLE
   END #CLASS
+  CLASS
+    NAME "Zone d'utilité publique 1 - Constructible"
+    EXPRESSION /1511/
+    STYLE
+        COLOR  157 159 162
+    END #STYLE
+  END #CLASS
+  CLASS
+    NAME "Zone d'utilité publique 2 - À constructibilité restreinte"
+    EXPRESSION /1611/
+    STYLE
+        COLOR 138 200 101
+    END #STYLE
+    STYLE
+        SYMBOL "hatchsymbol"
+        SIZE 6
+        COLOR 157 159 162
+        WIDTH 3
+    END #STYLE
+  END #CLASS
   CLASS
     NAME "Zone de verdure"
     EXPRESSION /1603|1613/
```

Comment:
```suggestion
    EXPRESSION {1603,1613}
```

Check here : https://mapserver.org/mapfile/expressions.html#list-expressions

Reply:
thanks! I wanted to do that anyway! 👍 
Still, I merge this PR to apply the changes to production. I'll push another PR later in the afternoon with the cleaned mapfile.